docs/pages/docs/getting-started.mdx

```diff
@@ -23,7 +23,7 @@ To see more examples and starters, have a look at the [example directory on GitH
 
 Turborepo was designed to be incrementally adopted. Adding it to an existing monorepo takes just a few minutes.
 
-Turborepo works with [Yarn v1](https://classic.yarnpkg.com/lang/en/), [NPM](https://npmjs.com), and [PNPM](https://pnpm.io/) workspaces. The `turbo` CLI works on the following operating systems.
+Turborepo works with [Yarn v1](https://classic.yarnpkg.com/lang/en/), [NPM](https://npmjs.com), and [PNPM](https://pnpm.io/) workspaces and [Node](https://nodejs.org/en/) v14 or higher. The `turbo` CLI works on the following operating systems.
```

Comment:
Turbo is written in Go and so this is not entirely accurate. `turbo` doesn't even need Node.js installed to function. 

`create-turbo` likely needs Node 12 or 14 to work is my guess, and the starter examples likely need the same. Choosing NPM package manager also would require npm > 7 (the first to support workpsaces).

I think this is too much information for "getting started" so I would rather enforce it in the NPM `package.json`

Reply:
Ah right, thanks for that clarification. That makes sense. The project created by `create-turbo` needs Node 14 to function based on my testing of running `npx create-turbo@latest` on different Node versions and then running `npm run build` in the turbopreo. 

I'll update this PR.